scripts/setup/delete-reinitialize-database

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env sh
+
+set -e
+set -x
+
+if [ "$(id -u)" -ne "0" ]; then
+  >&2 echo "$0 must be run as root."
+  exit 1
+fi
+
+cat <<EOF
+This script will completely wipe your database, including all users, chat
```

Comment:
This warning message is similar to https://github.com/zulip/zulip/blob/1dc741b963593cc4750b7a7e6994ce94b41e7345/scripts/setup/postgres-init-db#L17-L30. Maybe they should be deduplicated? Maybe the `-gt 200` threshold can be changed by setting an env var? The "It will not modify your existing /etc/zulip ..." is informative, maybe it should be added the postgres-init-db instead.

Reply:
yeah, deduplicating sounds good.